pytype/tools/analyze_project/main.py

```diff
@@ -74,6 +74,8 @@ def main():
   logging.info('Source tree:\n%s',
                importlab.output.formatted_deps_list(import_graph))
   tool_utils.makedirs_or_die(conf.output, 'Could not create output directory')
+  with open(conf.output + '/.gitignore', 'w') as f:
```

Comment:
Please use `os.path.join(conf.output, 'gitignore')` instead of `conf.output + '/.gitignore'`. (You'll also have to import os at the top of the file.) Using os.path.join avoids hard-coding a platform-specific separator and generates the correct path regardless of whether conf.output ends with a /.

Reply:
Thanks for your time @rchen152, changes are updated!

Reply:
Thank you! I'll import this change once the tests pass. (You'll see the PR being closed, rather than merged, but you'll be credited as the author in the commit history.)